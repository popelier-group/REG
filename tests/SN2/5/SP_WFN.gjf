# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.44359900   -0.00008300    0.00023400
 H                 -0.74651100   -0.88016600   -0.54782800
 H                 -0.74659400    0.91459200   -0.48785900
 H                 -0.74635600   -0.03471400    1.03648700
 Br                 1.59601500    0.00000000    0.00000400
 Cl                -3.38514700    0.00003700   -0.00011200

frame.wfn
