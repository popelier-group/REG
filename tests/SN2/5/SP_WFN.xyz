
6

C       -0.332713    0.005855    0.043519
H       -0.528402   -0.538971    0.955410
H       -0.759749   -0.475089   -0.824239
H       -0.610873    1.046962    0.117042
Br        1.689438   -0.028581   -0.220569
Cl       -3.249119    0.054842    0.424153
