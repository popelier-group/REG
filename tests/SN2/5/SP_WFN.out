 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.5/Gau-22931.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.5/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     22932.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.4436   -0.00008   0.00023 
 H                    -0.74651  -0.88017  -0.54783 
 H                    -0.74659   0.91459  -0.48786 
 H                    -0.74636  -0.03471   1.03649 
 Br                    1.59602   0.        0. 
 Cl                   -3.38515   0.00004  -0.00011 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.443599   -0.000083    0.000234
      2          1           0       -0.746511   -0.880166   -0.547828
      3          1           0       -0.746594    0.914592   -0.487859
      4          1           0       -0.746356   -0.034714    1.036487
      5         35           0        1.596015    0.000000    0.000004
      6         17           0       -3.385147    0.000037   -0.000112
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.080127   0.000000
     3  H    1.080126   1.795760   0.000000
     4  H    1.080130   1.795785   1.795776   0.000000
     5  Br   2.039614   2.561687   2.561700   2.561680   0.000000
     6  Cl   2.941548   2.834987   2.834832   2.835307   4.981162
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.332713    0.005855    0.043519
      2          1           0       -0.528402   -0.538971    0.955410
      3          1           0       -0.759749   -0.475089   -0.824239
      4          1           0       -0.610873    1.046962    0.117042
      5         35           0        1.689438   -0.028581   -0.220569
      6         17           0       -3.249119    0.054842    0.424153
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         155.4994077           0.8320191           0.8320190
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       176.9565119154 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  9.18D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 EnCoef did   100 forward-backward iterations
 SCF Done:  E(RB3LYP) =  -3074.39434793     A.U. after   12 cycles
            NFock= 12  Conv=0.13D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.68501-101.25882 -62.33805 -56.15786 -56.15473
 Alpha  occ. eigenvalues --  -56.15473 -10.07819  -9.17200  -8.54743  -6.93068
 Alpha  occ. eigenvalues --   -6.93007  -6.93007  -6.37708  -6.36675  -6.36675
 Alpha  occ. eigenvalues --   -2.48826  -2.48533  -2.48533  -2.47714  -2.47714
 Alpha  occ. eigenvalues --   -0.65812  -0.53829  -0.50993  -0.29929  -0.29928
 Alpha  occ. eigenvalues --   -0.23913  -0.15383  -0.15383  -0.05965  -0.05867
 Alpha  occ. eigenvalues --   -0.05867
 Alpha virt. eigenvalues --    0.09989   0.11054   0.14146   0.14146   0.16275
 Alpha virt. eigenvalues --    0.17457   0.18090   0.18091   0.21490   0.21982
 Alpha virt. eigenvalues --    0.21983   0.24332   0.24332   0.24683   0.31144
 Alpha virt. eigenvalues --    0.31145   0.31715   0.34265   0.40644   0.50338
 Alpha virt. eigenvalues --    0.52456   0.52456   0.54192   0.54192   0.67272
 Alpha virt. eigenvalues --    0.67682   0.67683   0.69010   0.74947   0.74947
 Alpha virt. eigenvalues --    0.75956   0.77506   0.77507   0.82885   0.82885
 Alpha virt. eigenvalues --    0.84446   0.95517   1.08647   1.14693   1.14693
 Alpha virt. eigenvalues --    1.15185   1.15186   1.16357   1.23398   1.23399
 Alpha virt. eigenvalues --    1.26131   1.56671   1.67610   1.67611   1.81290
 Alpha virt. eigenvalues --    1.85064   1.85064   2.08195   2.12760   2.12761
 Alpha virt. eigenvalues --    2.29295   2.31018   2.31019   2.52165   2.52165
 Alpha virt. eigenvalues --    2.56589   2.60586   2.64656   2.64657   2.68820
 Alpha virt. eigenvalues --    2.68820   3.12958   3.15359   3.86002   3.86003
 Alpha virt. eigenvalues --    4.28154   4.28154   4.37299   4.37299   4.66032
 Alpha virt. eigenvalues --    6.73848   7.69556   7.69556   7.79861   9.87977
 Alpha virt. eigenvalues --   24.04578  26.08156  26.08158  26.21320  48.03172
 Alpha virt. eigenvalues --  215.84648 289.89558 289.89558 290.016441020.81009
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.068741   0.412252   0.412264   0.412224   0.094500  -0.059737
     2  H    0.412252   0.529889  -0.035744  -0.035731  -0.056313   0.006599
     3  H    0.412264  -0.035744   0.529865  -0.035735  -0.056309   0.006599
     4  H    0.412224  -0.035731  -0.035735   0.529937  -0.056319   0.006601
     5  Br   0.094500  -0.056313  -0.056309  -0.056319  35.379880   0.008146
     6  Cl  -0.059737   0.006599   0.006599   0.006601   0.008146  17.915094
 Mulliken charges:
               1
     1  C   -0.340243
     2  H    0.179048
     3  H    0.179060
     4  H    0.179021
     5  Br  -0.313584
     6  Cl  -0.883303
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.196887
     5  Br  -0.313584
     6  Cl  -0.883303
 Electronic spatial extent (au):  <R**2>=           1170.0297
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             10.4105    Y=             -0.1754    Z=             -1.3589  Tot=             10.5002
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -95.5667   YY=            -43.8002   ZZ=            -44.6679
   XY=              0.8740   XZ=              6.7598   YZ=             -0.1141
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -34.2218   YY=             17.5448   ZZ=             16.6770
   XY=              0.8740   XZ=              6.7598   YZ=             -0.1141
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=            221.6373  YYY=             -0.2738  ZZZ=             -9.2915  XYY=             22.7027
  XXY=             -2.9928  XXZ=            -23.0305  XZZ=             25.2264  YZZ=             -1.2863
  YYZ=             -2.8694  XYZ=              0.2282
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1969.5254 YYYY=            -70.4190 ZZZZ=            -94.6132 XXXY=             20.0072
 XXXZ=            154.5091 YYYX=             11.5816 YYYZ=             -1.5090 ZZZX=             94.5557
 ZZZY=             -1.7958 XXYY=           -262.2456 XXZZ=           -274.0170 YYZZ=            -27.4901
 XXYZ=             -1.4267 YYXZ=             31.1207 ZZXY=              4.6548
 N-N= 1.769565119154D+02 E-N=-7.705337555487D+03  KE= 3.070011565036D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE525\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-0.443599,-0.000083,0.000234\H,0,-0.746511,-0.880166,-0.547828\H,0
 ,-0.746594,0.914592,-0.487859\H,0,-0.746356,-0.034714,1.036487\Br,0,1.
 596015,0.,0.000004\Cl,0,-3.385147,0.000037,-0.000112\\Version=ES64L-G1
 6RevC.01\State=1-A\HF=-3074.3943479\RMSD=1.254e-09\Dipole=4.131109,-0.
 0000974,0.0002874\Quadrupole=-26.1101122,13.0550517,13.0550605,0.00045
 61,-0.001419,0.0000026\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 SCIENCE AND PEACE WILL TRIUMPH OVER IGNORANCE AND WAR

                          -- PASTEUR
 Job cpu time:       0 days  0 hours  0 minutes 28.0 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 15.0 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:34:09 2020.
